packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -480,11 +499,16 @@
             padding-inline-start: 16px;
             padding-inline-end: 16px;
         }
-        @include mobile-or-tablet-screen {
-            height: unset;
+        @include tablet-screen {
             padding-inline-start: 0;
             padding-inline-end: 0;
         }
+
+        @include mobile {
```

Comment:
The break point `mobile-screen` is new one. That should work fine here 👀 
Is that not working fine for you ? 😬

Reply:
it was a simple merge conflict its resolved 👍